cg/meta/workflow/balsamic.py

```diff
@@ -256,8 +250,9 @@ def get_verified_bed(self, sample_data: dict, panel_bed: Path) -> Optional[str]:
         - When multiple samples have different parameters
         - When bed file required for analysis, but is not set or cannot be retrieved.
         """
-        application_types = set([v["application_type"].lower() for k, v in sample_data.items()])
-        target_beds = set([v["target_bed"] for k, v in sample_data.items()])
+        application_types = {v["application_type"].lower() for k, v in sample_data.items()}
```

Comment:
mmm k,v , is that not illegal with one letter variable names 😸

Reply:
They are in a generator though, so you never need to guess what they are, and actually you never care again once generator is finished